app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    masks_to_buy = 0
+    no_vaccinated = 0
+    for visitor in friends:
+        try:
+            cafe.visit_cafe(visitor)
+        except VaccineError:
```

Comment:
Consider handling specific exceptions (`NotVaccinatedError` and `OutdatedVaccineError`) separately instead of the general `VaccineError`. This will allow you to provide more specific feedback or actions based on the exact issue.